### PR TITLE
feat(ngff zarr): support loading version 0.4

### PR DESCRIPTION
**Pull request by PaulHax**
BREAKING CHANGE: Removes support for consolidated metadata in NGFF zarr files.

Currently no support for time and channel dimensions.

### CODE REVIEW

File: test/zarrTest.js [hunk 1]
```diff
@@ -1,355 +1,118 @@
...
-  const image1 = await image.scaleLargestImage(1)
-  t.deepEqual(image1.imageType, imageTypeBaseline, 'image1 imageType')
-  t.deepEqual(image1.size, [240, 147], 'image1 size')
-  t.deepEqual(image1.data.length, 105840, 'image1 data length')
+  t.deepEqual(image.imageType, imageTypeBaseline, 'image type set')
+  t.equal(image.name, 'testimage', 'image name set')
+  t.deepEqual(image.origin, [0.5, 0.5], 'image origin set')
+  t.deepEqual(image.spacing, [1, 1], 'image spacing set')
+  t.deepEqual(image.size, [64, 64], 'image size set')
+  t.equal(image.data.length, 4096, 'image data lenght set')
```
Comment (PaulHax) [inline test/zarrTest.js:22]: length sp =_)

File: test/zarrTest.js [hunk 2]
```diff
@@ -1,355 +1,118 @@
...
 }
 
-test('Test ConsolidatedMetadataStore', async t => {
+test('Test MetadataStore', async t => {
```
Comment (PaulHax) [inline test/zarrTest.js:25]: Zarr Store

File: src/IO/ZarrStore.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+   * Zarr HTTP store.
+   */
+
+  constructor(url) {
+    this.url = url
+  }
+
+  async getItem(item) {
+    const itemUrl = `${this.url.href}/${item}`
+    const { data } = await axios.get(itemUrl, {
```
Comment (PaulHax) [inline src/IO/ZarrStore.js:19]: should alwasy fetch as arraybuffer

File: src/IO/ZarrStore.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import axios from 'axios'
+
+const isMetadata = item =>
+  ['.zattrs', '.zgroup', '.zarray'].some(knownMetadataFile =>
+    item.endsWith(knownMetadataFile)
+  )
+
+class ZarrStore {
```
Comment (PaulHax) [inline src/IO/ZarrStore.js:8]: HTTPStore

File: src/IO/ZarrStore.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    this.decoder = new TextDecoder()
+  }
+
+  toJson(data) {
+    return JSON.parse(this.decoder.decode(data))
+  }
+
+  async getItem(item) {
+    const data = await this.store.getItem(item)
+    console.log(data)
```
Comment (thewtex) [inline src/IO/ZarrStore.js:18]: debug code

**Review by thewtex — APPROVED** [review]
@PaulHax awesome, Paul!!

Debug code noted inline that can be removed in follow-up.

**Pull request merged**